snap/get_version

```diff
@@ -0,0 +1,114 @@
+#!/bin/dash
```

Comment:
OOC why /bin/dash and not /bin/sh

Reply:
Local testing enforcement, forgot to remove it, nice spot!